api/routes/book_groups/bookGroups.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response } from 'express';
+import { UserInfoRequest } from '../../middleware/requests';
+import pool from '../../configDB/config';
+
+export const getBookGroups = async (request: UserInfoRequest, response: Response) => {
+  try {
+    const userId: number = request.user;
+    const {
+      rows,
+    } = await pool.query('SELECT * FROM book_groups WHERE (is_public=TRUE) OR (user_id=$1) ORDER BY id ASC', [userId]);
+    response.status(200).json(rows);
+  } catch (err) {
+    response.status(500).json(err);
+  }
+};
+
+export const getBookGroup = async (req: Request, res: Response) => {
+  await pool
+    .query('SELECT * FROM book_groups WHERE (id=$1)', [req.params.id])
+    .then(results => res.status(200).json(results.rows))

```

Comment:
Dlaczego then jak korzystamy wyżej z await?

Reply:
To przez przypadek wrzuciłem - zacząłem pisać i nie skończyłem :)